docs-master/Installation.md

```diff
@@ -73,6 +73,12 @@ npm install @nozbe/with-observables
               pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi', :modular_headers => true
               ```
 
+          3. Finally, add this line:
+
+             ```ruby
+             pod 'FMDB/SQLCipher', :modular_headers => true
```

Comment:
I don't think this should be required. Seems appropriate to me to add to docs-master/Advanced a new .md for how to set up encryption. Alternatively, just put this at the end and specify that this is optional